test/jasmine/tests/bar_test.js

```diff
@@ -623,13 +871,13 @@ describe('A bar plot', function() {
                 [1, 2, 3, 4], [1, 2, 3, 4]]);
             assertPointField(cd, 'y', [
                 [1, 2, 3, 4], [4, 4, 4, 4],
-                [-1, -3, -2, -4], [4, -4, -5, -6]]);
+                [-1, -3, -2, -4], [4, -3.25, -5, -6]]);
```

Comment:
I wanted to have a small negative bar, so that the label `"outside text"` doesn't fit inside the bar and has to be drawn below the bar (as opposed to the positive bar case that has outside texts drawn above the bar).